### PR TITLE
[user manual] add functions and operators

### PR DESCRIPTION
**Pull request by jude-zhu**

### CODE REVIEW

File: docs/manul_doc/Functions_and_Operators/Comparison_Functions_and_Operators.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+
+| **  Name  **   | ** Description  **   |
```
Comment (darionyaphet) [inline docs/manul_doc/Functions_and_Operators/Comparison_Functions_and_Operators.md]: format error ?

File: docs/manul_doc/Functions_and_Operators/Functions_and_Operator_Reference.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+| ** Name   **   | ** Description   **   |
```
Comment (darionyaphet) [inline docs/manul_doc/Functions_and_Operators/Functions_and_Operator_Reference.md]: ditto

File: docs/manul_doc/Functions_and_Operators/Logical_Operators.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+| **  Name  **   | ** Description  **   |
```
Comment (darionyaphet) [inline docs/manul_doc/Functions_and_Operators/Logical_Operators.md]: ditto

File: docs/manul_doc/Functions_and_Operators/Comparison_Functions_and_Operators.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+|   <     |   Less than operator    | 
+|   <=     |   Less than or equal operator    | 
+|   !=    |   Not equal operator    | 
+
+
+Comparison operations result in a value of _true_ and _false_.
+
+* ==
+Equal. String comparisons are case-sensitive. Values of different type are not equal.
+```
```
Comment (laura-ding) [inline docs/manul_doc/Functions_and_Operators/Comparison_Functions_and_Operators.md]: should have an empty line before **```**

File: docs/manul_doc/Functions_and_Operators/Logical_Operators.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+| (-(1)&&true) |
+================
+|         true |
+----------------
+```
+
+* !
+Logical NOT:
+```
+nebula> nebula> YIELD !(-1);
```
Comment (laura-ding) [inline docs/manul_doc/Functions_and_Operators/Logical_Operators.md]: why has double `nebula>`

File: docs/manul_doc/Functions_and_Operators/Comparison_Functions_and_Operators.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+nebula> YIELD '2' == 2;
+============
+| ("2"==2) |
+============
+|    false |
+------------
+```
+
+* >
+Less than：
```
Comment (laura-ding) [inline docs/manul_doc/Functions_and_Operators/Comparison_Functions_and_Operators.md]: Greater than 
and the example also need to modify

File: docs/manul_doc/Functions_and_Operators/Logical_Operators.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
+|  Name    |  Description    | 
+|:----|:----:|
+|   &&     |   Logical AND     | 
+|   !    |   Logical NOT     | 
+|   \|\|   |   Logical OR   | 
+
+
+In nGQL, nonzero numbers are evaluted to _true_. The precedence of the operators refer to [Operator Precedence] (./Operator_Precendence.md)
```
Comment (darionyaphet) [inline docs/manul_doc/Functions_and_Operators/Logical_Operators.md]: link is broken

File: docs/manul_doc/Functions_and_Operators/Operator_Precedence.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+The following list shows the precedence of nGQL operators, in descending order. Operators on a line have the same precedence.
+```
```
Comment (darionyaphet) [inline docs/manul_doc/Functions_and_Operators/Operator_Precedence.md]: add a empty line would be better ?

File: docs/manul_doc/Functions_and_Operators/Operator_Precedence.md [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+
+```
+!
+- (unary minus)
+*, /, %
+-, +
+== , >=, >, <=, <, <>, !=
+&&
+||
+= (assignment)
```
Comment (whitewum) [inline docs/manul_doc/Functions_and_Operators/Operator_Precedence.md]: can assignment be used t

File: docs/manul_doc/Functions_and_Operators/Functions_and_Operator_Reference.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
+|  Name    |  Description    |
```
Comment (whitewum) [inline docs/manul_doc/Functions_and_Operators/Functions_and_Operator_Reference.md]: if not supported yet, comment out.

Comment (jude-zhu) [reply docs/manul_doc/Functions_and_Operators/Functions_and_Operator_Reference.md]: functions in this table are all supported.

File: docs/manul_doc/Functions_and_Operators/Operator_Precedence.md (hunk 9 above)
Comment (jude-zhu) [reply docs/manul_doc/Functions_and_Operators/Operator_Precedence.md]: only works when assigning "go from" results at this moment, e.g. "$a=GO FROM 201 OVER like; GO FROM $a.id OVER select YIELD $^.student.name;"

File: docs/manul_doc/Functions_and_Operators/Operators.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+| +   | Addition operator   | 
+| !   | Logical NOT   | 
+| *   | Multiplication operator   | 
+| -   | Change the sign of the argument   | 
+
+
+Example:
+
+```
+nebula> $a=GO FROM 201 OVER like; GO FROM $a.id OVER select YIELD $^.student.name;
```
Comment (whitewum) [inline docs/manul_doc/Functions_and_Operators/Operators.md]: e.g., where ($.name >= "aa") || ($.age < 18)  to explain >=, ()

**Comment by laura-ding** [comment]
jenkins go

File: docs/manul_doc/Language_Structure/Identifier_Case_Sensitivity.md [hunk 12]
```diff
@@ -0,0 +1,19 @@
+In Nebula Graph, Identifiers are case-sensitive.
```
Comment (Amber1990Zhang) [inline docs/manul_doc/Language_Structure/Identifier_Case_Sensitivity.md]: identifiers or Identifiers？

**Pull request merged**